modules.py

```diff
@@ -83,10 +108,10 @@ def save(self, name=None):
 
         if name is None:
```

Comment:
This code structure could be simplified by a line like:

name = "{}.fits".format(name is not None and name or self.name)
hdu.writeto('name, clobber=True)
print >> sys.stdout, 'Saving {}'.format(name)


and then just one 'save' chunk is needed.  The above line sets the value of the variable 'name' to the value of the variable 'name' if the current value of 'name' is not None other-wise name=self.name and then